bus-board/src/modules/IJsonBus.ts

```diff
@@ -0,0 +1,29 @@
+export interface IJsonBus {

```

Comment:
The I before the interface name isn't the standard anymore in react, you can just name them normally.